packages/rn-tester/js/examples/Pressable/PressableExample.js

```diff
@@ -250,6 +256,31 @@ function PressableDisabled() {
   );
 }
 
+function PressableFocusBlurEvents() {
+  const [lastEvent, setLastEvent] = useState('');
+
+  return (
+    <View testID="pressable_hit_slop">
+      <View style={[styles.row, styles.centered]}>
+        <Pressable
+          onFocus={() => {
+            console.log('focused lol');
+            setLastEvent('received focus event');
+          }}
+          onBlur={() => {
+            setLastEvent('received blur event');
+          }}
+          testID="pressable_focus_blur_button">
+          <Text style={styles.text}>Use keyboard to move focus to me</Text>
```

Comment:
Since the the onFocus logic was moved to `View`, I'm doubting this works with `Text`? We may need to separately add this logic for `Text` as well...